azurerm/internal/services/web/app_service.go

```diff
@@ -1812,6 +1992,55 @@ func flattenAppServiceStorageAccounts(input map[string]*web.AzureStorageInfoValu
 	return results
 }
 
+func expandAppServiceAutoHealRules(input map[string]interface{}) (*web.AutoHealRules, error) {
+	triggerVals := input["triggers"].(map[string]interface{})
+
+	autoHeal := web.AutoHealRules{}
+
+	triggers := web.AutoHealTriggers{}
+	slowRequests := triggerVals["slow_requets"].(map[string]interface{})
```

Comment:
This will crash if this is nil and not map[string]interface{}.  Since this is actually a list that needs to be unpacked, this would be the case. If you use `expandAppServiceLogs` in this file as an example, you can see one way to handle this.